CMakeLists.txt

```diff
@@ -33,20 +33,19 @@ set(CMAKE_CXX_FLAGS_RELEASE "")
 set(CMAKE_CXX_FLAGS_MINSIZEREL "")
 
 
-if (CMAKE_CL_64 STREQUAL "1")
-  message(STATUS "Bitness: 64 bit ('${CMAKE_CL_64}')")
-  set(IS_64_BIT TRUE)
-  set(IS_32_BIT FALSE)
-else()
-  message(STATUS "Bitness: 32 bit ('${CMAKE_CL_64}')")
-  set(IS_64_BIT TRUE)
-  set(IS_32_BIT FALSE)
-endif()
-
 set(LIBRARIES "")
 
 # Windows - 32 and 64 bit
 if (WIN32)
+  if (CMAKE_CL_64 STREQUAL "1")
+    message(STATUS "Bitness: 64 bit ('${CMAKE_CL_64}')")
+    set(IS_64_BIT TRUE)
+    set(IS_32_BIT FALSE)
+  else()
+    message(STATUS "Bitness: 32 bit ('${CMAKE_CL_64}')")
+    set(IS_64_BIT TRUE)
+    set(IS_32_BIT FALSE)
```

Comment:
I believe its an error that IS_32_BIT is set to FALSE here aswell, but since this was also the case before, I guess it works anyway so we can leave it